sdk/webpubsub/Microsoft.Azure.WebJobs.Extensions.WebPubSub/CHANGELOG.md

```diff
@@ -1,12 +1,8 @@
 # Release History
 
-## 1.8.0-beta.1 (Unreleased)
-
+## 1.8.0 (Unreleased)
```

Comment:
Add a date if we're planning to release soon?

Reply:
Added